src/main/kotlin/org/rust/ide/intentions/WrapLambdaExprIntention.kt

```diff
@@ -32,9 +35,11 @@ class WrapLambdaExprIntention : PsiElementBaseIntentionAction() {
     override fun invoke(project: Project, editor: Editor, element: PsiElement) {
         val ctx = findContext(element) ?: return
         val lambdaBody = ctx.lambdaExpr.expr ?: return
-        val bodyStr = ctx.lambdaExpr.expr?.text ?: return
+        var bodyStr = ctx.lambdaExpr.expr?.text ?: return
+        bodyStr = "\n$bodyStr\n"
```

Comment:
I think this is fine: formatter wont change `|x| { x }` into 

```
|x | {
    s
}
```

because it is Ok with both variants. But when you want to add braces to lambda, you almost always want the second formatting.